src/packages/admin-ui-components/src/detail-panel/fields/relationship-field.tsx

```diff
@@ -68,10 +69,12 @@ export const RelationshipField = ({
 		}
 	);
 
+	const [inputValue, onInputChange] = useState('');
+
 	const options = (data?.result ?? []).map<SelectOption>((item): SelectOption => {
 		const label = relatedEntity.summaryField || relatedEntity.primaryKeyField;
 		return { label: item[label], value: item[relatedEntity.primaryKeyField] };
-	});
+	}).filter(item => inputValue?.toLowerCase().length > 0 ? item.label?.toLowerCase().includes(inputValue.toLowerCase()) : true);
```

Comment:
Implement really dumb client side filtering for relationships. A better solution might be to adjust the filter params passed to the server, like in the relationship filter. This solution just applies a client side match to filter results when you type in the combo box